src/test/end-to-end/shared/shared-functions.ts

```diff
@@ -1,3 +1,12 @@
+
+export function monthFormatted(month) {
```

Comment:
monthFormatted() and dayFormatted() methods are not used anymore. Danny replaced them with the generic padFormatted() method in master. You can removed them.

Reply:
Done :)